float32 FigurHeight
float32 FigurOffset
float32 FaceAngle
